pkg/kubelet/nodelease/controller.go

```diff
@@ -210,25 +224,6 @@ func (c *controller) newLease(base *coordinationv1.Lease) *coordinationv1.Lease
 	}
 	lease.Spec.RenewTime = &metav1.MicroTime{Time: c.clock.Now()}
 
-	// Setting owner reference needs node's UID. Note that it is different from
-	// kubelet.nodeRef.UID. When lease is initially created, it is possible that
-	// the connection between master and node is not ready yet. So try to set
-	// owner reference every time when renewing the lease, until successful.
-	if len(lease.OwnerReferences) == 0 {
```

Comment:
When updating, `base` is from `c.latestLease`, which already has non-empty owner references

Reply:
It shouldn't happen in healthy situation, but yes - in theory it's possible.

That's why I think I'm against this change.

Reply:
Ack. I will keep the existing behavior in #95428. Thanks

/close